### PR TITLE
Profile settings

### PR DESCRIPTION
**Pull request by robhillman97**
Adds:

- log-in/out
- Register (redirect to webpage)
- Change Moodle server
- Change LRS server
- 
to profile page

 Logged In

Logged Out

Change Moodle

Change LRS

### CODE REVIEW

File: Assets/MirageXR/Tests/NewUI/MoodleServersView.cs [hunk 1]
```diff
@@ -1,11 +1,70 @@
...
+        
     }
+
+    private void OnClickWEKIT() {
+        ChangeServerDomain("https://learn.wekit-ecs.com");
+    }
+
+    private void OnClickARETE()
+    {
+        ChangeServerDomain("https://arete.ucd.ie");
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/MoodleServersView.cs]: move `"https://learn.wekit-ecs.com"` and `"https://arete.ucd.ie"` to `DBManager` as a const values

File: Assets/MirageXR/Tests/NewUI/ProfileView.cs [hunk 2]
```diff
@@ -3,25 +3,191 @@
...
+        _txtLogout.text = $"You are already logged in,\n<b>{DBManager.username}</b>";
+        _toggleRemember.isOn = DBManager.rememberUser;
+        _inputFieldUserName.text = string.Empty;
+        _inputFieldPassword.text = string.Empty;
+        _inputFieldUserName.ResetValidation();
+        _inputFieldPassword.ResetValidation();
+        // _learningRecordStoreDropdown.value = DBManager.publicCurrentLearningRecordStore;
+
+        UpdateConectedLRS(DBManager.publicCurrentLearningRecordStore);
+        }
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/ProfileView.cs:130]:
```suggestion
}
```

File: Assets/MirageXR/Player/Scripts/Moodle/DBManager.cs [hunk 3]
```diff
@@ -140,6 +142,12 @@ public static string domain
             set => _domain.Value = value;
         }
 
+        public static int publicCurrentLearningRecordStore
```
Comment (cawke) [inline Assets/MirageXR/Player/Scripts/Moodle/DBManager.cs]: reaplace with `Enum`

File: Assets/MirageXR/Tests/NewUI/ProfileView.cs [hunk 4]
```diff
@@ -3,25 +3,191 @@
...
+        PopupsViewer.Instance.Show(_LRSPrefab);
+    }
+
+    private void UpdateConnectedServerText() 
+    {
+        _txtConnectedServer.text = DBManager.domain;
+    }
+
+
+    private void UpdateConectedLRS(int publicCurrentLearningRecordStore)
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/ProfileView.cs]: `int publicCurrentLearningRecordStore` reaplace with `Enum`

File: Assets/MirageXR/Tests/NewUI/SelectLRS.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    private void OnClickWEKIT() {
+        ChangeRecordStore(0);
+    }
+
+    private void OnClickARETE()
+    {
+        ChangeRecordStore(1);
+    }
+
+    private void ChangeRecordStore(int selectedLearningRecordStore)
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/SelectLRS.cs]: `int selectedLearningRecordStore` reaplace with `Enum`

**Review by cawke — CHANGES_REQUESTED** [review]

**Comment by cawke** [comment]
@robhillman97 if you are ready to review, just press the `re-request review` button
image

**Comment by fwild** [comment]
Can we finalise this?

**Review by cawke — APPROVED** [review]
ty

**Pull request merged**